tools/ThermoRawFileParser/thermo_converter.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="thermo_raw_file_converter" name="Thermo" version="1.2.3">
+<tool id="thermo_raw_file_converter" name="Thermo" version="1.2.3.5">
```

Comment:
can we change this to `1.2.3+galaxy0`.  The suffix is somehow our conda-build string.